aten/src/ATen/native/native_functions.yaml

```diff
@@ -1363,6 +1363,18 @@
     CPU: batch_norm_update_stats_cpu
     CUDA: batch_norm_update_stats_cuda
 
+- func: native_layer_norm(Tensor input, Tensor? weight, Tensor? bias, int[] normalized_shape, float eps) -> (Tensor, Tensor, Tensor)
+  dispatch:
+    CPU: LayerNormForwardCPU
```

Comment:
can you use the standard camel case, i.e. `layer_norm_forward_cpu` throughout this PR?